src/cr/cube/collator.py

```diff
@@ -331,4 +331,65 @@ def _display_order(self):
         in the `"exclude": [...]` array of the order transform. Base elements appear in
         value-sorted order within their grouping.
         """
+        hidden_idxs = self._hidden_idxs
+        return tuple(
+            idx
+            for idx in (
+                self._top_subtotal_idxs
+                + self._top_exclusion_idxs
```

Comment:
"exclusion" sounds right in the construct "exclusion from sort", but on its own it might suggest that the elements are somehow excluded completely, which is the first thing I thought. I then went back to the docstring and figured out what it meant. Maybe consider a slightly different approach to naming? Something like `_top_not_sorted_idxs`?

Reply:
Yeah, that might be an idea. The exclusion language came in right at the start when this was specced, as apparently the users' language for this concept and I didn't question it too much. It's probably too much to change at this point because we've consistently applied that term from the schema and validation all the way through to here, so probably more work than we can afford at this point and I'm not sure there really is a purely intuitively-descriptive name for the concept.